test/e2e/cloud_provider_kubevirt/common.go

```diff
@@ -0,0 +1,32 @@
+package cloud_provider_kubevirt
```

Comment:
let's not have utils, common we know how this endup...
Can we do the package thing that we did at other projects ? 

`test/e2e/naming/namespce.go`
`test/e2e/resource/operations.go`

Reply:
[Done](https://github.com/kubevirt/cloud-provider-kubevirt/compare/f30c1d886eeb9025476a68a588a6adf37b5ac6f3..05b2ee2a05ec846999a6063fe9431ee5dc831297)